src/main/java/io/lighty/yang/validator/YangContextFactory.java

```diff
@@ -163,7 +163,7 @@ private static String readFile(final String path) throws IOException {
         final List<File> yangFiles = new ArrayList<>();
         File[] files = dir.listFiles();
         if (files == null) {
-            return null;
+            return Collections.EMPTY_LIST;
```

Comment:
Collections.emptyList() or List.of() (JDK 11)

Reply:
Done